Sources/Extensions/UIKit/Layout.swift

```diff
@@ -44,15 +44,49 @@ public extension UIView {
             return []
         }
 
-        let constrains: [NSLayoutConstraint] = [
+        let constraints: [NSLayoutConstraint] = [
             topAnchor.constraint(equalTo: superview.layoutMarginsGuide.topAnchor),
             leadingAnchor.constraint(equalTo: superview.layoutMarginsGuide.leadingAnchor),
             trailingAnchor.constraint(equalTo: superview.layoutMarginsGuide.trailingAnchor),
             bottomAnchor.constraint(equalTo: superview.layoutMarginsGuide.bottomAnchor),
         ]
 
-        NSLayoutConstraint.activate(constrains)
+        NSLayoutConstraint.activate(constraints)
```

Comment:
why do we activate and also returning the cons?

Reply:
I guess it can be useful to keep a reference to them if you want to replace them after. I think it's ok to have the option to do this